projects/sbb-esta/angular-business/tooltip/src/tooltip/tooltip.ts

```diff
@@ -0,0 +1,482 @@
+/* tslint:disable:no-non-null-assertion */
```

Comment:
This can be disabled directly in tslint.json (It's already disabled in the bazel branch)